torcheeg/transforms/any/baseline.py

```diff
@@ -47,9 +47,42 @@ def apply(self, eeg: any, **kwargs) -> any:
             'baseline'].shape == eeg.shape, f'The shape of baseline signals ({kwargs["baseline"].shape}) need to be consistent with the input signal ({eeg.shape}). Did you forget to add apply_to_baseline=True to the transforms before BaselineRemoval so that these transforms are applied to the baseline signal simultaneously?'
         return eeg - kwargs['baseline']
 
+   
+
+class BaselineCorrection(EEGTransform):
+    r'''
+    A transform method to subtract the mean of baseline signal from EEG.
+    
+    TorchEEG recommends using this class in online_transform for higher processing speed. Even though, this class is also supported in offline_transform. Usually, the baseline needs the same transformation as the experimental signal, please add :obj:`apply_to_baseline=True` to all transforms before this operation to ensure that the transformation is performed on the baseline signal
+
+    .. code-block:: python
+
+        from torcheeg import transforms
+
+        t = transforms.BaseCorrection()
+
+        t(eeg=np.random.randn(32, 128), baseline=np.random.randn(32, 128))['eeg'].shape
+        >>> (32,128)
+    
+    .. automethod:: __call__
+    '''
+    def __init__(self):
+        super(BaselineCorrection, self).__init__(apply_to_baseline=False)
+        
+
+    def __call__(self, *args, eeg: any, baseline= None, **kwargs) :
+        return super().__call__(*args, eeg=eeg, baseline=baseline, **kwargs)
+
+
+    def apply(self, eeg, **kwargs) -> any:
+         assert len(eeg.shape) == 2 and  len(kwargs['baseline'].shape) ==2, "Make sure EEG and its baseline in 2D shape. "
```

Comment:
This implementation does not actually meet expectations. In many works, baseline removal is applied to features, corresponding to the average of high-order tensors along the segment dimension, e.g., [1]. This is also why we did not directly use the implementation from EEGLab.

Yang Y, Wu Q, Qiu M, et al. Emotion recognition from multi-channel EEG through parallel convolutional recurrent neural network[C]//2018 international joint conference on neural networks (IJCNN). IEEE, 2018: 1-7.

Reply:
Due to the complex related components, you can temporarily keep this part of the code. I will make the modifications.